iis-wf/iis-wf-primary/src/main/resources/eu/dnetlib/iis/wf/primary/main/oozie_app/workflow.xml

```diff
@@ -490,6 +490,10 @@
                     <name>output_faults</name>
                     <value>${workingDir}/primary_import/faults</value>
                 </property>
+                <property>
+                    <name>output_report</name>
+                    <value>${workingDir}/report</value>
```

Comment:
For now this would work but considering we are going to produce counters for various importer modules we need more fine-grained reports directory structure along with joiners. We are going to produce report datastores in several places within importer subworkflow (metadata importer which is already covered but also content importer, dataset etc) and we wound need to join them before providing them to uber workflow (caller).